databricks_offline_store.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Databricks Offline Store Setup
+description: Using Databricks as your offline store.
+published: true
+---
+
+## Databricks Workspace Prerequisites
+To use Databricks as the offline store for Chalk, Chalk needs access to a Databricks workspace
+(requires “Pricing tier” to be Premium or higher, as required functionality (Databricks SQL) is
```

Comment:
@melrchen not sure if these smart quotes are intentional but they spook me

Reply:
lol i can sub out the quotes